### PR TITLE
prevent "Job for docker.service canceled" error on startup

### LINKED ISSUE #1424: Job for docker.service canceled flake
```
  {
    "id": "/subscriptions/225e02bc-43d0-43d1-a01a-17e584a4ef69/resourceGroups/e2e-upgrade-jim-minter-1407-wshawq/providers/Microsoft.Resources/deployments/azuredeploy/operations/EE57228B898B50BF",
    "operationId": "EE57228B898B50BF",
    "properties": {
      "provisioningState": "Failed",
      "timestamp": "2019-04-03T06:00:32.1918821Z",
      "serviceRequestId": "8ace60ec-b578-4966-a4fe-7ac2b759f7b6",
      "statusCode": "Conflict",
      "statusMessage": {
        "error": {
          "code": "ResourceDeploymentFailure",
          "details": [
            {
              "code": "VMExtensionProvisioningError",
              "message": "VM has reported a failure when processing extension 'cse'. Error message: \"Enable failed: failed to execute command: command terminated with exit status=1\
[stdout]\
\
[stderr]\
+ grep /var/lib/docker /etc/fstab\
+ systemctl stop docker.service\
Job for docker.service canceled.\
\"."
            }
          ],
          "message": "The resource operation completed with terminal provisioning state 'Failed'."
        },
        "status": "Failed"
      },
      "targetResource": {
        "id": "/subscriptions/225e02bc-43d0-43d1-a01a-17e584a4ef69/resourceGroups/e2e-upgrade-jim-minter-1407-wshawq/providers/Microsoft.Compute/virtualMachineScaleSets/ss-infra-1554271042",
        "resourceName": "ss-infra-1554271042",
        "resourceType": "Microsoft.Compute/virtualMachineScaleSets"
      }
    }
  },
```

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
prevent "Job for docker.service canceled" error on startup
```

fixes #1424

### CODE REVIEW

**Comment by jim-minter** [comment]
@mjudeikis @ehashman @kwoodson wdyt?

Am working out with MSFT if we need to backport this to 4.x.

File: pkg/arm/v5/data/master-startup.sh [hunk 1]
```diff
@@ -12,6 +14,7 @@ if ! grep /var/lib/docker /etc/fstab; then
 }
 EOF
   systemctl start docker.service
+  systemctl start docker-cleanup.timer
```
Comment (y-cote) [inline pkg/arm/v5/data/master-startup.sh]: do you not have to restart systemctl stop docker-cleanup.service as well here?

Comment (ehashman) [reply pkg/arm/v5/data/master-startup.sh]: I believe starting the timer will start the service, but I had the same question.

Comment (jim-minter) [reply pkg/arm/v5/data/master-startup.sh]: the service is oneshot.  The timer starts it each time it goes off.

Comment (y-cote) [reply pkg/arm/v5/data/master-startup.sh]: Ok now I get it, thanks.

**Comment by jim-minter** [comment]
/hold cancel

backporting to 4.3, starting tests again

**Comment by jim-minter** [comment]
thanks @mjudeikis

**Pull request merged**